modules/file-system/pre-existing-network-storage/scripts/mount.sh

```diff
@@ -40,3 +28,11 @@ if ! grep -q "${MOUNT_IDENTIFIER}" /etc/fstab; then
 else
 	echo "Skipping editing /etc/fstab as entry already exists"
 fi
+
+if ! findmnt --source "${SERVER_IP}":"${REMOTE_MOUNT_WITH_SLASH}" --target "${LOCAL_MOUNT}" &>/dev/null; then
+	echo "Mounting --target ${LOCAL_MOUNT} from fstab"
+	mkdir -p "${LOCAL_MOUNT}"
+	mount --target "${LOCAL_MOUNT}"
```

Comment:
Maybe just check for a return value > 0 here. 
See mount return codes here: https://linux.die.net/man/8/mount

Reply:
This has been resolved by using `set -e` on this script.